lib/src/page_list_viewport_gestures.dart

```diff
@@ -288,35 +305,190 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
-  /// momentum simulation to be aborted.
-  static double kViewportReAdjustmentMinTranslationDistance = 60;
+  /// {@macro velocity_increase}
+  static const double largeTranslationNormalVelocityMultiplier = 0.85;
+
+  /// Velocity multiplier that should be applied when the user releases an arbitrary direction
+  /// panning motion (not locked axis), and the content goes ballistic.
+  ///
+  /// This value is unit-less and should be multiplied by a velocity that's measured in pixels
+  /// per second.
+  /// Speed up the diagonal ballistic simulation.
+  static const double diagonalLaunchVelocityMultiplier = 0.7;
+
+  /// Default velocity multiplier that should be applied when the user lifts
+  /// their finger after a panning motion when the content goes ballistic.
+  ///
+  /// This value is unit-less and should be multiplied by a velocity that's measured in pixels
+  /// per second.
+  static const double defaultVelocityMultiplier = 1.0;
+
+  /// Increase the drag coefficient of the ballistic simulation.
+  ///
+  /// Higher drag coefficient means that the simulation launched after user lifts
+  /// their finger will deccelerate faster.
+  /// The drag deccelaration term in the simulation is -d/dt(v) = dragCoefficient * v.
+  static const double defaultDragMultiplier = 1.0;
+
+  /// Minimal translation distance required for a gesture to be considered for axis locking.
+  ///
+  /// After the user has panned more than this distance, the gesture will be locked
+  /// if it is close enough to the horizontal or vertical axis as defined in
+  /// [horizontalAxisLockAngle] and [verticalAxisLockAngle].
+  /// Artificially increase the distance to prevent axis locking for tiny gestures.
+  /// Note that this depends on the rate at which the gestures are sampled.
+  static const double minAxisLockingTranslationDistance = 2.0;
+
+  /// Angle w.r.t. the horizontal axis for a gesture to be locked to the horizontal axis.
+  ///
+  /// {@template axis_locking_angles}
+  /// The angle defines a window around the axis, in which the gesture
+  /// will be locked to the axis.
+  /// The larger this angle, the easier gestures will be locked to the axis.
+  /// The angle is measured in radians.
+  /// {@endtemplate}
+  static const double horizontalAxisLockAngle = math.pi / 12;
+
+  /// Angle w.r.t. the vertical axis for a gesture to be locked to the vertical axis.
+  ///
+  /// {@macro axis_locking_angles}
+  static const double verticalAxisLockAngle = math.pi / 4;
+
+  /// Maximal time between any two scrolling gestures for them to be considered for viewport scrolling acceleration
+  ///
+  /// Scrolls which are repeated frequently and are in the same direction should cause the viewport
+  /// to scroll faster and faster with each consequtive swiping input.
+  /// This is called repeated swipe (or scroll) acceleration
+  static const Duration timeBetweenRepeatedScrollGestures = Duration(milliseconds: 1000);
```

Comment:
The name of this property doesn't seem to match the description of its meaning. Perhaps it should be called `maxDurationForRepeatGesturesToAcceleratePanning`